docs/source/array-api.rst

```diff
@@ -30,7 +30,7 @@ Top level functions
    argtopk
    argwhere
    around
-   array
+   Array
```

Comment:
Was this giving a warning before? I think the intention of this entry is to document the top-level `dask.array.array` function, which is different from the `dask.array.Array` class constructor:

```python
In [1]: import dask.array as da

In [2]: da.array
Out[2]: <function dask.array.routines.array(x, dtype=None, ndmin=None, *, like=None)>

In [3]: da.Array
Out[3]: dask.array.core.Array
```

Reply:
Thanks for clarifying, I wasn't sure about the intention. Yes, the warning that it was throwing was:

```
/Users/koverholt/Repos/dask/docs/source/array-api.rst:9: WARNING: autosummary: stub file not found 'dask.array.array'. Check your autosummary_generate setting.
```

I thought it was unintentional, but with your comment I am not sure why this stub is the only one complaining. I'll look into this a bit more, but let me know if you have any insight based on the above warning.

Reply:
It looks like there is no `docs/build/html/generated/dask.array.array.html` where the other stubs are generated.

Is there something different about https://github.com/dask/dask/blob/a5aecac8313fea30c5503f534c71f325b1775b9c/dask/array/routines.py#L44-L53 that would result in `dask.array.array` being the only function with no generated stub?

Reply:
I reverted my change and set it back to `array`, and although it throws a warning on my machine, it seems to be working fine in RTD docs build:

https://readthedocs.org/api/v2/build/15413948.txt (the warnings shown in that build log will go away after https://github.com/dask/distributed/pull/5549 is merged)

https://dask--8432.org.readthedocs.build/en/8432/generated/dask.array.array.html

So this PR should be ready to go now.